etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy

```diff
@@ -136,10 +136,11 @@ class TreatmentObsMigrator extends ObsMigrator {
     }
 
     def void migrateProphylaxesPlan() {
+        // This is actually historical information.
+        // But it's in the Plan section on the OpenMRS Intake form for some reason.
```

Comment:
Why are you assuming this is historical information?  The history or prophylaxes is collected in section 7 of the HIV EMR Intake form (as have already been migrated), this is about migrating the "plan" information, which is collected in Section 20 of the HIV EMR Intake.

What suggests to you the fact that this is historical?  The "stop" and "continue".  I would assume that it's possible that patients are already on prophylaxes upon intake, but this is still a "plan" whether or not to continue that treatment?

Reply:
Oh, that's a fair point. It's "stopped" and "continued" (in the past tense), which is what got me mixed up. I think I've spent too much time with this piece of form and it's made my head spin.

Reply:
That's fair enough re: head spinning! :)